languages/translations/es-ES.json

```diff
@@ -417,24 +417,24 @@
         },
         "roundabout turn": {
             "default": {
-                "default": "En la rotonda siga {modifier}",
-                "name": "En la rotonda siga {modifier} por {way_name}",
-                "destination": "En la rotonda siga {modifier} hacia {destination}"
+                "default": "Siga {modifier}",
+                "name": "Siga {modifier} en {way_name}",
+                "destination": "Siga {modifier} hacia {destination}"
             },
             "left": {
-                "default": "En la rotonda gira a la izquierda",
-                "name": "En la rotonda gira a la izquierda por {way_name}",
-                "destination": "En la rotonda gira a la izquierda hacia {destination}"
+                "default": "Gire a la izquierda",
+                "name": "Gire a la izquierda en {way_name}",
+                "destination": "Gire a la izquierda hacia {destination}"
             },
             "right": {
-                "default": "En la rotonda gira a la derecha",
-                "name": "En la rotonda gira a la derecha por {way_name}",
-                "destination": "En la rotonda gira a la derecha hacia {destination}"
+                "default": "Gire a la derecha",
+                "name": "Gire a la derecha en {way_name}",
+                "destination": "Gire a la derecha hacia {destination}"
             },
             "straight": {
-                "default": "En la rotonda continúa recto",
-                "name": "En la rotonda continúa recto por {way_name}",
-                "destination": "En la rotonda continúa recto hacia {destination}"
+                "default": "Continúe recto",
+                "name": "Continúa recto por {way_name}",
```

Comment:
Mixed tenses here: `Continúe` `Continúa` - was there an issue with the find and replace?

Reply:
Yes, I think so. Transifex’s find and replace is not as reliable as one would expect. 😞

Reply:
I did find-and-replace on the `es` localization but not the `es-ES` localization, which looked at a glance to already have been standardized. I’ll fix up the `es-ES` localization.

Reply:
Fixed in d1335e6dce662fd57d1563a2eea9253d4c267ee9.